### ISSUE TITLE
build: name 'git_repository' is not defined

### ISSUE THREAD

**Issue by sergey-safarov** [hiveco/conntrack_exporter#12]
On master
```
bash-5.1# make
bazel build -c dbg //:conntrack_exporter
ERROR: /tmp/conntrack_exporter/WORKSPACE:1:1: name 'git_repository' is not defined
ERROR: error loading package '': Encountered error while reading extension file 'repositories.bzl': no such package '@prometheus_cpp//': error loading package 'external': Could not load //external package
INFO: Elapsed time: 0.074s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
make: *** [Makefile:4: build] Error 1
bash-5.1# 
```

Related description
> git_repository is no longer a native rule. You need to include it at the top of your WORKSPACE with:
```
load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```
[Link](https://stackoverflow.com/questions/55190124/solve-dependency-issue-when-using-grpc-cpp-in-bazel)

**Comment by dbendelman** [comment]
@sergey-safarov thanks for reporting. As stated in the readme, this has only been tested on Ubuntu 16.04 and Bazel v0.7.0. If you get it to work on higher versions feel free to open a PR and I'll be happy to merge.

**Comment by dbendelman** [comment]
Closed by mistake

**Comment by dbendelman** [comment]
Fixed in #14
